democracy_club/urls.py

```diff
@@ -27,11 +27,6 @@
         name="auth_logout",
     ),
     path("", HomeView.as_view(), name="home"),
-    path(
-        "data_apis/index/",
-        TemplateView.as_view(template_name="data_apis/index.html"),
-        name="data_and_apis",
-    ),
```

Comment:
Fixes #604 ?

I think this was a short term URL, so we might not need a redirect for this one?

Reply:
That's correct, it doesn't exist anymore - but wasn't sure if we needed to redirect to address the sentry errors - even if just temporarily. Shall I remove?

Reply:
I think the sentry error was just from the old view, so this is also ok! :+1: